apps/gitness/src/pages-v2/repo/labels/label-form-container.tsx

```diff
@@ -14,52 +14,31 @@ export const RepoLabelFormContainer = () => {
   const { spaceId, repoId, labelId } = useParams<PathParams>()
   const navigate = useNavigate()
 
-  const { isLoading: isDataLoading, repo_ref } = useFullFillLabelStore({
-    query: labelId,
-    enabled: !!labelId
-  })
+  const { repo_ref } = useFullFillLabelStore({ query: labelId, enabled: !!labelId })
+
+  const onFormCancel = () => navigate(routes.toRepoLabels({ spaceId, repoId }))
 
   const {
     mutate,
```

Comment:
Not an issue, just a "while you're there" kinda thing. It's usually a good idea to alias the `mutate` so that it's obvious what it does when calling it. If it's called `mutate`, you have to scroll back up to the top to figure out what it does. If it's called `saveRepoLabel`, it's immediately obvious.